src/Core/Framework/MessageQueue/Command/ScheduleScheduledTaskCommand.php

```diff
@@ -0,0 +1,74 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Framework\MessageQueue\Command;
+
+use Shopware\Core\Framework\Adapter\Console\ShopwareStyle;
+use Shopware\Core\Framework\Context;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\MessageQueue\ScheduledTask\Registry\TaskRegistry;
+use Shopware\Core\Framework\MessageQueue\ScheduledTask\ScheduledTaskDefinition;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+#[AsCommand(
+    name: 'scheduled-task:schedule',
+    description: 'Schedule a scheduled task',
+)]
+#[Package('framework')]
+class ScheduleScheduledTaskCommand extends Command
+{
+    private const STATUSES_SUCCESS = [
+        ScheduledTaskDefinition::STATUS_SCHEDULED,
+        ScheduledTaskDefinition::STATUS_SKIPPED,
+    ];
+
+    private const STATUSES_FORCE_NEEDED = [
+        ScheduledTaskDefinition::STATUS_QUEUED,
+        ScheduledTaskDefinition::STATUS_RUNNING,
+    ];
+
+    /**
+     * @internal
+     */
+    public function __construct(private readonly TaskRegistry $taskRegistry)
+    {
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this->addArgument('taskName', InputArgument::REQUIRED, 'Scheduled task name like log_entry.cleanup');
+        $this->addOption('force', 'f', null, 'Force the scheduling of the scheduled task');
+        $this->addOption('immediately', 'i', null, 'Set the next execution time to now');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new ShopwareStyle($input, $output);
+
+        $taskName = $input->getArgument('taskName');
+        $immediately = (bool) $input->getOption('immediately');
+        $force = $input->getOption('force');
```

Comment:
why is the bool casting not necessary for the "force" option? :thinking:

Reply:
PHPStan didn't complain, but I added it in https://github.com/shopware/shopware/pull/11670/commits/0fefaac44a6b1b6f386c2288463acd1f51c99787